.jenkins/pytorch/win-test.sh

```diff
@@ -40,6 +40,10 @@ if [ -n "$CIRCLE_PULL_REQUEST" ]; then
   file_diff_from_base "$DETERMINE_FROM"
 fi
 
+if [[ "${CIRCLE_JOB}" == *11.1* ]]; then
```

Comment:
Same here

Reply:
This I've tested overnight--I believe this works!